content/messaging/agent-chat-api/rtm-reference/index.mdx

```diff
@@ -3845,10 +3845,11 @@ Informs that a chat was transferred to a different group or to an Agent.
 | Object | Notes                        |
 | ------ | ---------------------------- |
 | `thread_id`      | present if the chat is active                        |
-| `transferred_to` [^1] | IDs of groups and Agents the chat is now assigned to |
+| `transferred_to` **\*** | IDs of groups and Agents the chat is now assigned to |
 | `queue`          | present if the chat is queued after the transfer     |
 
-[^1]: One of the following combinations of ids is possible:
+**\*)**
+One of the following combinations of ids is possible:
 
 - both `group_ids` and `agent_ids`, when the chat was transferred to a group, and an agent was assigned immediately (even if the chat was transferred within the same group),
 - only `group_ids`, when the chat was transferred to a group and no agents were available, queue will be present,
```

Comment:
```suggestion
- Only `group_ids`. When a chat was transferred to a group and no Agents were available; the queue will be present.
```

Reply:
Line 3855: 
Only `agent_ids`. When a chat was transferred directly to an Agent; there'll be no queue.